src/picongpu/submit/taurus-tud/k80_profile.tpl

```diff
@@ -46,11 +46,12 @@ TBG_nodes="$(( ( TBG_tasks + TBG_gpusPerNode -1 ) / TBG_gpusPerNode))"
 # Sets batch job's name
 #SBATCH --job-name=!TBG_jobName
 #SBATCH --nodes=!TBG_nodes
-#SBATCH --ntasks-per-node=!TBG_mpiTasksPerNode
+#SBATCH --ntasks=!TBG_tasks
+#SBATCH --mincpus=!TBG_mpiTasksPerNode
 #SBATCH --cpus-per-task=!TBG_coresPerGPU
-#SBATCH --mem-per-cpu=3000
+#SBATCH --mem-per-cpu=2583
 #SBATCH --gres=gpu:!TBG_gpusPerNode
-# send me a mail on (b)egin, (e)nd, (a)bortion
+# send me a mail on BEGIN, END, FAIL, REQUEUE or ALL
```

Comment:
If you do not select a mail type, `sbatch` will not submit the job. Even though, `slurm` support flag lists, it does not support an empty list.